internal/sirius/search_users.go

```diff
@@ -67,23 +64,21 @@ func (c *Client) SearchUsers(ctx Context, search string) ([]User, error) {
 		return nil, newStatusError(resp)
 	}
 
-	var v apiUserList
+	var v []apiUser
```

Comment:
Why move from a more descriptive name to a less descriptive one?

Reply:
change made in a previous PR and rebased into ours